SourceKitStressTester/Sources/Common/Message.swift

```diff
@@ -122,13 +118,27 @@ public struct Page: Codable {
     return number - 1
   }
 
+  public init() {
+    self.number = 1
+    self.count = 1
+  }
```

Comment:
Could default the arguments on the overload below instead to save duplicating it.

Reply:
Good point, thanks